api/lib/domain/models/CertificationChallenge.js

```diff
@@ -6,13 +6,36 @@ class CertificationChallenge {
     challengeId,
     courseId,
     competenceId,
+    isNeutralized,
   } = {}) {
     this.id = id;
     this.associatedSkillName = associatedSkillName;
     this.associatedSkillId = associatedSkillId;
     this.challengeId = challengeId;
     this.competenceId = competenceId;
     this.courseId = courseId;
+    this.isNeutralized = isNeutralized;
+  }
+
+  static new({
```

Comment:
suggestion: 
 on a l'habitude d'utiliser 'from' dans le code. new est reservé en javascript, ce qui peut préter à confusion
```suggestion
  static from({
```

Reply:
je lève le pouce vis à vis de l'usage de new que je pense qu'il faut éviter

Reply:
Hum, le `from` j'aime bien l'utiliser qu'on en instancie des objets à partir d'autres objets et qu'il y a un peu d'algo/traitement et de décapsulation à faire : 

`const assessmentResult.from(answers)` (exemple bidon qui n'existe pas en vrai)

ici j'instancie mon objet à partir de données brutes (comme si j'utilisais un bête constructeur) mais ce que je cherche à faire, c'est d'indiquer que la `CertificationChallenge` est en cours de création au niveau de son cycle de vie. En particulier, si elle est en cours de création, elle est forcément en `isNeutralized = false`. On ne peut pas créer une CertificationChallenge déjà neutralisée, ça n'a pas de sens fonctionnellement. 
Est-ce que `create` vous irait ?

Reply:
oui create c'est ok pour moi, je souhaite juste qu'on évite le mot new

Reply:
Si tu regarde le code, tu ne pars pas vraiment de donnees brut mais d'un objet a peu pres equivalent que tu cree à la volée (facile en js 😁 )
Cela dit, je comprends ta démarche, et le build irais aussi. L'important est d'eviter le `new` 👍